pkg/controller/daemon/daemoncontroller_test.go

```diff
@@ -58,17 +58,22 @@ var (
 	noScheduleTaints      = []v1.Taint{{Key: "dedicated", Value: "user1", Effect: "NoSchedule"}}
 )
 
+func nowPointer() *metav1.Time {
```

Comment:
I think it dilutes the metav1 package... there's no `time.NowP()` function in the stdlib, for example